terraform/bod_bastion_ec2.tf

```diff
@@ -44,3 +44,19 @@ resource "aws_instance" "bod_bastion" {
     },
   )
 }
+
+# Provision the bastion EC2 instance via Ansible
+module "bod_bastion_ansible_provisioner" {
+  source = "github.com/cloudposse/terraform-null-ansible"
+
+  arguments = [
+    "--user=${var.remote_ssh_user}",
+    "--ssh-common-args='-o StrictHostKeyChecking=no'",
```

Comment:
Alphabetize args:
```suggestion
    "--ssh-common-args='-o StrictHostKeyChecking=no'",
    "--user=${var.remote_ssh_user}",
```

Reply:
I simply copied the CyHy bastion's block and made necessary changes so if you are amenable I would like to do this in a separate PR that alphabetizes the values feeding all of the Ansible provisioner modules (both the SSH arguments and the environment variables where applicable).

Reply:
Sure, that's fine by me. 👍